lib/modules/datasource/hermit/readme.md

```diff
@@ -0,0 +1,18 @@
+[Hermit](https://cashapp.github.io/hermit/) by default looks up packages from the open source project [https://github.com/cashapp/hermit-packages](https://github.com/cashapp/hermit-packages).
+
+Hermit supports [private packages](https://cashapp.github.io/hermit/packaging/private/). To make your private packages works with Renovate, you will need to do the following.
+
+1. perform `hermit search --json` with your private Hermit distribution and save the file to `index.json`
+2. make a Github release in your private packages repository named `index` with the asset `index.json` generated in step 1.
+3. setup a CI pipeline to repeat step 1 & 2 on new commits to the private packages repository.
```

Comment:
```suggestion
1. perform `hermit search --json` with your private Hermit distribution and save the file to `index.json`
1. make a GitHub release in your private packages repository named `index` with the asset `index.json` generated in step 1.
1. setup a CI pipeline to repeat step 1 & 2 on new commits to the private packages repository.
```

- The rendered Markdown numbers each item for us. We can just use `1.` to create a numbered list.
- GitHub is a proper noun, so should be capitalized.